lib/project.rb

```diff
@@ -170,6 +172,14 @@ def init_workspace
         @logger.warn('Workspace path already exists')
       end
       @logger.info("Workspace path: #{@workspace_path}")
+
+      begin
+        File.delete("#{@config['workspace_path']}/latest")
+      rescue Errno::ENOENT
+        # firts run, no symlink to delete
+      end
```

Comment:
No need for deletion; `FileUtils.ln_sf` overwrites.

Reply:
Does not work for the directory